### PR TITLE
hotfix: 20231026

### PR DESCRIPTION
**Pull request by lorux0**
## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### <samp>🤖 Generated by Copilot at 4738134</samp>

This pull request adds support for showing the amount of emotes and wearables owned by a user in the backpack and avatar editor HUDs. It also improves the performance and consistency of the wearables catalog service by using shortened urns that exclude the token ids. Additionally, it updates some animation assets and scripts to use the new format and values introduced in Unity 2020.3, and modifies the `DCLTexture` class to handle different types of textures. The changes affect several files in the `unity-renderer/Assets` folder, such as `EmotesRequestWeb.cs`, `LambdasWearablesCatalogService.cs`, `WearableResponse.cs`, `DCLTexture.cs`, `EmoteCardComponentView.cs`, `WearableGridController.cs`, `WearableGridItemComponentView.cs`, `ExtendedUrnParser.cs`, `WearableItemResolver.cs`, `EmoteCardComponentModel.cs`, `EmotesCustomizationComponentController.cs`, `BackpackEditorHUDController.cs`, `BackpackEmotesSectionController.cs`, `WearableGridItemModel.cs`, and `NFTTypesIcons.asset`.

### CODE REVIEW

**Review by anicalbano — APPROVED** [review]
🟢 Hotfix reviewed by QA

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEmotesSectionController.cs [hunk 1]
```diff
@@ -105,6 +105,21 @@ async UniTaskVoid LoadEmotesAsync(CancellationToken ct = default)
                     allEmotes.AddRange(embeddedEmotesSo.GetAllEmotes());
                     allEmotes.AddRange(await emotesCatalogService.RequestOwnedEmotesAsync(userProfileBridge.GetOwn().userId, ct) ?? Array.Empty<WearableItem>());
 
+                    Dictionary<string, WearableItem> consolidatedEmotes = new Dictionary<string, WearableItem>();
```
Comment (popuz) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEmotesSectionController.cs:108]: if this method called often, then I would suggest to cache/Clear dictionary or use Unity pools 

```
using (DictionaryPool<string, WearableItem>.Get(out  Dictionary<string, WearableItem> consolidatedEmotes))
{
foreach (var emote in allEmotes) {...}
}
```

**Review by popuz — APPROVED** [review]
LGTM. minor optimization suggestion

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEmotesSectionController.cs (hunk 1 above)
Comment (lorux0) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEmotesSectionController.cs:108]: Will do it on `dev`

**Pull request merged**